python-numpy-basics/Classes_and_Objects.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>Use <a href="https://www.freecodecamp.org/news/how-to-format-code-in-markdown/" target="_blank">Python Syntax Highlighting</a> for general format of the a class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
yes, encorporated that